compiler/qsc_rir/src/passes/build_dominator_graph/tests.rs

```diff
@@ -51,7 +37,8 @@ fn test_dominator_graph_single_block_dominates_itself() {
         .insert(BlockId(0), Block(vec![Instruction::Return]));
 
     remap_block_ids(&mut program);
```

Comment:
Does it make sense to extract theses three calls into a common fn? All of the tests use the same setup.

Reply:
I can do that here and in the other test file as well.